docs/book/src/multiversion-tutorial/testdata/project/test/e2e/e2e_test.go

```diff
@@ -110,7 +110,7 @@ var _ = Describe("Manager", Ordered, func() {
 					"pods", controllerPodName, "-o", "jsonpath={.status.phase}",
 					"-n", namespace,
 				)
-				g.Expect(utils.Run(cmd)).To(BeEquivalentTo("Running"), "Incorrect controller-manager pod status")
+				g.Expect(utils.Run(cmd)).To(Equal("Running"), "Incorrect controller-manager pod status")
```

Comment:
Just for curiosity,
what is the diff between `BeEquivalentTo ` and `Equal` do you know?

Reply:
Equal checks for equality while BeEquivalentTo does some type conversion, like between string and []byte.  Unfortunately, it still prints out the byte array representation.   Equals doesn't allow a byte array to be compared to a string.